hail/src/main/scala/is/hail/io/gen/LoadGen.scala

```diff
@@ -20,18 +21,18 @@ case class GenResult(file: String, nSamples: Int, nVariants: Int, rdd: RDD[(Anno
 
 object LoadGen {
   def apply(
-    genFile: String,
-    sampleFile: String,
-    sc: SparkContext,
-    rgBc: Option[Broadcast[ReferenceGenome]],
-    nPartitions: Option[Int] = None,
-    tolerance: Double = 0.02,
-    chromosome: Option[String] = None,
-    contigRecoding: Map[String, String] = Map.empty[String, String],
-    skipInvalidLoci: Boolean = false): GenResult = {
-
-    val hConf = sc.hadoopConfiguration
-    val sampleIds = LoadBgen.readSampleFile(hConf, sampleFile)
+   genFile: String,
+   sampleFile: String,
+   sc: SparkContext,
+   fs: FS,
+   rgBc: Option[Broadcast[ReferenceGenome]],
+   nPartitions: Option[Int] = None,
+   tolerance: Double = 0.02,
+   chromosome: Option[String] = None,
+   contigRecoding: Map[String, String] = Map.empty[String, String],
+   skipInvalidLoci: Boolean = false): GenResult = {
```

Comment:
did your editor do this, these lines were consistent with our previous formatting?

Reply:
I did this because this alignment, vertically inline with the function declaration, wasn't consistent with function declarations elsewhere in this file, and more globally. Look at readGenLine, the next function down. I restored it anyway.

Intellij adds much more significant indentation by default: pins to the left parenthesis.

Reply:
I'm not sure why it's not showing up as identical. I copy-pasted from the master version, then added Fs:FS. At this point it's an Intellij formatting issue.

Reply:
Probably tabs vs spaces